components/kyma-environment-broker/internal/process/staged_manager.go

```diff
@@ -151,6 +151,7 @@ func (m *StagedManager) Execute(operationID string) (time.Duration, error) {
 
 			// the step needs a retry
 			if when > 0 {
+				logStep.Warnf("retrying step by restarting the operation in %d s", when.Seconds())
```

Comment:
internal/process/staged_manager.go:154:5: (*github.com/sirupsen/logrus.Entry).Warnf format %d has arg when.Seconds() of wrong type float64